.rubocop.yml

```diff
@@ -5,9 +5,13 @@ AllCops:
     - config.ru
     - db/schema.rb
     - vendor/**/*
+    - lib/ckan/v26/depaginator.rb
```

Comment:
Why do we need to do this?

Reply:
It's a very tough class to make readable (and not currently worth our while to do a really significant refactor) and every change rubocop flagged would make it harder to read.

Reply:
Primarily around the heredocs.